pages/Types.html

```diff
@@ -9,7 +9,7 @@
 ol ul li { font-size: 1em !important; }
 ol ul { margin-left: 1.5em !important; }
 </style>
-<p>JavaScript provides several built-in datatypes. In addition to those, this page documents virtual types like Selectors, enhanced pseudo-types like Events and all and everything you wanted to know about Functions.
+<p>JavaScript provides several built-in datatypes. In addition to those, this page documents virtual types like Selectors, enhanced pseudo-types like Events and some concepts you need to know about Functions. If you want to study in depth these concepts, take a look at <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">MDN</a>.
```

Comment:
@AurelioDeRosa, thanks for this! Could you change "in depth these concepts" to "these concepts in depth"?